src/main/java/gr/grnet/pccapi/client/eoscportal/EOSCPortalClient.java

```diff
@@ -1,6 +1,5 @@
-package gr.grnet.pccapi.client;
+package gr.grnet.pccapi.client.eoscportal;
```

Comment:
why do we use eoscPortal?
What does this has to do with the pcc ?

Reply:
We use the following endpoint to retrieve the available Eosc-Portal scientific domains:
`https://api.eosc-portal.eu/vocabulary/byType`